src/main/java/blackjack/domain/card/Score.java

```diff
@@ -25,14 +25,26 @@ public Score subtract(int value) {
         return new Score(score - value);
     }
 
-    public boolean isBlackJack() {
+    public boolean isBlackjack() {
         return score == BLACK_JACK;
     }
 
     public boolean isBurst() {
         return score > BLACK_JACK;
     }
 
+    public boolean isNotBurst() {
+        return score <= BLACK_JACK;
+    }
+
+    public boolean isUnderBlackjack() {
+        return score < BLACK_JACK;
+    }
+
+    public boolean isNotBlackjack() {
+        return score != BLACK_JACK;
```

Comment:
```suggestion
        return !isBlackjack();
```
서로 반대 의미를 갖는 메서드를 만들 때에는 더 주요한 메서드를 다른 한쪽이 의존하도록 만들면 관리하기도 편할 것 같아요 :)

Reply:
오~~!! 관리하기가 더 편하겠네요! 수정하였습니다! :)